src/main/resources/templates/index.html

```diff
@@ -16,6 +16,10 @@
     <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
     <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+    
```

Comment:
Scripts should usually be at the top of an HTML document.
ref
https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup